go/client/cmd_device_remove.go

```diff
@@ -134,7 +138,11 @@ func NewCmdDeviceRemove(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 		Flags: []cli.Flag{
 			cli.BoolFlag{
 				Name:  "f, force",
-				Usage: "Override warning about removing the current device.",
+				Usage: "Force removal of the current device.",
+			},
+			cli.BoolFlag{
+				Name:  "last",
```

Comment:
I wonder if it makes sense for `--force` (or whatever we call it) and `--last` to imply that you're revoking the current device, without requiring you to specify the current device's ID in the positional args? Though in that case we probably want to give them long scary names?